extensions/wikia/Forum/ForumController.class.php

```diff
@@ -328,22 +325,22 @@ public function forumRelatedThreads() {
 			$messages = [ ];
 			$count = 0;
 			foreach ( $out as $key => $val ) {
-				if ( $title->getArticleId() == $val[ 'comment_id' ] ) {
+				if ( $title->getArticleId() == $val['comment_id'] ) {
 					continue;
 				}
 
-				$msg = WallMessage::newFromId( $val[ 'comment_id' ] );
+				$msg = WallMessage::newFromId( $val['comment_id'] );
 				if ( !empty( $msg ) ) {
 					$msg->load();
 
 					$message = [ 'message' => $msg ];
 
-					if ( !empty( $val[ 'last_child' ] ) ) {
-						$childMsg = WallMessage::newFromId( $val[ 'last_child' ] );
+					if ( !empty( $val['last_child'] ) ) {
+						$childMsg = WallMessage::newFromId( $val['last_child'] );
 
 						if ( !empty( $childMsg ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS134)